rust/processor/src/models/default_models/transactions.rs

```diff
@@ -186,13 +194,35 @@ impl Transaction {
             .timestamp
             .as_ref()
             .expect("Transaction timestamp doesn't exist!");
+
+        let (wsc, wsc_detail) = WriteSetChangeModel::from_write_set_changes(
+            &transaction_info.changes,
+            version,
+            block_height,
+            deprecated_tables,
+        );
+
+        if Transaction::skip_processing(deprecated_tables) {
```

Comment:
It's kinda messy that we tied the struct to the table name, because that's not always the case that 1 struct maps cleanly to 1 table. As we see here, `Transaction` actually parses a lot more other structs than just `transactions`. 

I liked @CapCap 's idea of having the filter only inside `insert_to_db`. Performance is not an issue because CPU is "cheap" compared to IO.

Reply:
agreed, it's really messy that we pass flags around every place. I misunderstood, I thought Max suggested to have this logic within the struct parsing but not DB writing. 

We should be able to get rid of this anyway once we have a parquet deployed or this could be replaced with a further upstream filter or transaction filter what max is working on?